websites/A/Amazon Kindle/metadata.json

```
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "202740603790819328",
+		"name": "aimslut"
+	},
+	"service": "Amazon Kindle",
+	"description": {
+		"en": "Read books online via Amazon Kindle"
```

Consider using one of the official descriptions instead
```suggestion
		"en": "Access and read millions of titles instantly and comfortably on your phone, tablet, or computer."
```